WordPress/src/main/java/org/wordpress/android/ui/pages/PagesActivity.kt

```diff
@@ -36,6 +42,12 @@ class PagesActivity : LocaleAwareActivity(),
     }
 
     private fun handleIntent(intent: Intent) {
+        if (intent.hasExtra(ARG_NOTIFICATION_TYPE)) {
+            val notificationType: NotificationType =
+                    intent.getSerializableExtra(ARG_NOTIFICATION_TYPE) as NotificationType
+            systemNotificationTracker.trackTappedNotification(notificationType)
```

Comment:
I'm getting an error here for `systemNotificationTracker` not being initialized. I think we are missing to declare this activity Injectable (something like `void inject(PagesActivity object);` in `AppComponent.java`) and actually inject it (in this activity onCreate something like `(application as WordPress).component().inject(this)`); let me know wdyt 😊

Reply:
🤦 fixed this in c23e3bf3b9